packages/create-keystone-app/README.md

```diff
@@ -6,15 +6,41 @@ title: Create Keystone App
 
 # Create Keystone App
 
-A cli for Keystone 5 to help generate starter apps.
+A CLI for Keystone 5 to help generate starter apps.
 
 ## Usage
 
+### Arguments
```

Comment:
Let's move the `Arguments` section to the end of this file - it's important info, but I'd prefer the actual command for running the CLI to sit above the fold where possible.

Reply:
Fixed in a6ef9afa